var/spack/repos/builtin/packages/fontconfig/package.py

```diff
@@ -35,6 +35,7 @@ class Fontconfig(AutotoolsPackage):
     version('2.11.1', 'e75e303b4f7756c2b16203a57ac87eba')
 
     depends_on('freetype')
+    depends_on('gperf', type='build')
```

Comment:
If you know this dependency is only needed for 2.12.2+, can you make that clear in the `depends_on`?
```python
depends_on('gperf', type='build', when='@2.12.2:')
```